### PR TITLE
Add -Xdump option to all SCM01 tests

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Related: https://github.com/eclipse/openj9/issues/8972#issuecomment-716523940

**Comment by Mesbah-Alam** [comment]
Test: Grinder/11584/console

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java [hunk 1]
```diff
@@ -253,7 +253,7 @@ public void execute(StfCoreExtension test, StfSharedClassesExtension sharedClass
 		String args = scTest.classArgs;
 		
 		// If we are running SharedClasses.SCM01.MultiThread
```
Comment (pshipton) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java:255]: Pls update the comment.

Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java]: Thanks for catching! Updated.

**Comment by pshipton** [comment]
Pls squash the commits, we don't need two.

**Comment by Mesbah-Alam** [comment]
The two commits have been squashed into one.

**Review by pshipton — APPROVED** [review]

**Pull request merged**